src/test/java/org/mitre/synthea/export/CCDAExporterTest.java

```diff
@@ -27,19 +33,51 @@
 import org.mitre.synthea.world.agents.PayerManager;
 import org.mitre.synthea.world.agents.Person;
 import org.mitre.synthea.world.geography.Location;
+import org.w3c.dom.Document;
+import org.w3c.dom.NodeList;
 
 /**
- * Uses Model Driven Health Tools (MDHT) to validate exported CCDA R2.1.
- * https://github.com/mdht/mdht-models
+ * This is a rudimentary check of the CCDA Export. It uses XPath expressions to ensure that the
```

Comment:
> It uses XPath expressions

I am so sorry. You have my condolences.

Reply:
I honestly don't find XPath to be all that bad. The worst part of writing this code was getting the expressions into java strings and ensuring that I didn't exceed the line length checkstyle rule.